ProjectManager/ProjectManager/Source/View/ToDoDetailView.swift

```diff
@@ -0,0 +1,111 @@
+        textField.layer.shadowColor = UIColor.systemGray2.cgColor
+        textField.layer.shadowOffset = CGSize(width: 0, height: 5)
+        textField.layer.shadowOpacity = 1
+        textField.layer.shadowRadius = 5.0
+        textField.leftView = UIView(frame: CGRect(x: 0.0, y: 0.0, width: 16.0, height: 0.0))
+        textField.leftViewMode = .always
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        
+        return textField
+    }()
+    
+    private let datePicker: UIDatePicker = {
+        let datePicker = UIDatePicker()
+        
+        datePicker.preferredDatePickerStyle = .wheels
+        datePicker.datePickerMode = .date
+        datePicker.isUserInteractionEnabled = true
+        datePicker.translatesAutoresizingMaskIntoConstraints = false
+        
+        return datePicker
+    }()
+    
+    private let bodyTextView: UITextView = {
+        let textView = UITextView()
+        
+        textView.font = UIFont.preferredFont(forTextStyle: .callout)
+        textView.textColor = .black
+        textView.backgroundColor = .systemBackground
+        textView.layer.masksToBounds = false
+        textView.layer.shadowColor = UIColor.systemGray2.cgColor
+        textView.layer.shadowOffset = CGSize(width: 0, height: 5)
+        textView.layer.shadowOpacity = 1
+        textView.layer.shadowRadius = 5.0
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return textView
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupView() {
+        addSubview(titleTextField)
+        addSubview(datePicker)
+        addSubview(bodyTextView)
+        
+        NSLayoutConstraint.activate([
+            titleTextField.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 12),
+            titleTextField.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -12),
+            titleTextField.topAnchor.constraint(equalTo: self.topAnchor),
+            titleTextField.heightAnchor.constraint(equalToConstant: 40),
+            datePicker.centerXAnchor.constraint(equalTo: self.centerXAnchor),
+            datePicker.topAnchor.constraint(equalTo: titleTextField.bottomAnchor, constant: 12),
+            bodyTextView.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 12),
+            bodyTextView.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -12),
+            bodyTextView.topAnchor.constraint(equalTo: datePicker.bottomAnchor, constant: 12),
+            bodyTextView.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: 12)
+        ])
+    }
+    
+    func validToDoTitle() -> Bool {
+        if titleTextField.text != .init() {
+            return true
+        }
+        return false
+    }
```

Comment:
[P5]
조건문 자체가 Boolean 타입이니 그대로 반환하면 될 것 같아요. 내부적으로 유효성을 검증하고 있지 않으니 `Valid`라는 이름을 사용하기에는 모자람이 있어보이니 구현을 보강하거나 이름을 변경하면 좋을 것 같아요.

```suggestion
    var hasTitle: Bool {
        return titleTextField.text?.isEmpty == false
    }
```

Reply:
라이언의 의견처럼 내부로 유효성을 검증하지 않고 단순히 `Title`의 text 유무를 판단하고 있기 때문에 참고하여 다음과 같이 수정했습니다!

```swift
    var hasTitle: Bool {
        guard let count = titleTextField.text?.split(separator: " ").count else { return false }
        
        return count < 1 ? false : true
    }
```

[refactor: validToDoTitle 메서드를 hasTitle 프로퍼티로 변경](https://github.com/yagom-academy/ios-project-manager/commit/3424281b3165f042bb3c24aed0a6f03de4064c7f)

Reply:
`return count < 1 ? false : true`는 `return count >= 1`과 같나요?
조건문 자체가 Boolean 타입이면 삼항 연산자를 사용하지 않아도 될 것 같네요.